pipeline_dp/combiners.py

```diff
@@ -178,22 +177,39 @@ def compute_metrics(self, accum: AccumulatorType) -> namedtuple:
         return MeanTuple(count=noisy_count, sum=noisy_sum, mean=noisy_mean)
 
 
+MetricsTuple = namedtuple('MetricsTuple',
```

Comment:
My idea is to create MetricsTuple in constructor of CompoundCombiner, namely

```
class CompoundCombiner.
   def __init__(self, ...)
       ...
       # here we find metrics_to_compute, e.g. ['count', 'mean'] 
       ...
       self._MetricsTuple = namedtuple("MetricsTuple", metrics_to_compute)
   
   def compute_metrics():
        # compute all metrics to output in list or tuple **metrics**
        return self._MetricsTuple(*metrics)
```
        
 That way we can even create metrics dynamically, e.g. percentile_90.

WDYT?